### PR TITLE
PDF cache

### PR DESCRIPTION
**Pull request by franticBIT**

### CODE REVIEW

File: middleware/checkCache.js [hunk 1]
```diff
@@ -5,25 +5,19 @@ const logTag = '[CheckCache]';
...
-  if (!req.query.format && /image\/webp/.test(accept)) {
-    logger.verbose(logTag, 'Client accepts webp');
-    req.query.format = 'webp';
-  }
-
   const queryHash = hash(req.query);
   req.fileHash = queryHash;
 
   try {
     if (cache.exists(queryHash)) {
```
Comment (tofixx) [inline middleware/checkCache.js]: cache.valid() should be used, maybe still not in develop

File: middleware/extractFilename.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 function extractFilename(response, fileParam) {
```
Comment (tofixx) [inline middleware/extractFilename.js]: is that used for images too?

File: routes/image.js [hunk 3]
```diff
@@ -14,7 +16,9 @@ const onlyDevelopment = require('../middleware/onlyDevelopment');
 router.get('/', checkQueryParams, checkEtag, checkCache, imageLoader, resize, convert, persist, function (req, res) {
   res.setHeader('Cache-Control', 'public, max-age=' + config.get('Caching.Expires'));
   res.setHeader('Expires', new Date(Date.now() + config.get('Caching.Expires')).toUTCString());
```
Comment (tofixx) [inline routes/image.js]: add filename

File: middleware/extractFilename.js (hunk 2 above)
Comment (franticBIT) [inline middleware/extractFilename.js]: Not at the moment. 😉

File: middleware/checkCache.js (hunk 1 above)
Comment (franticBIT) [inline middleware/checkCache.js]: Well, that will be merged then... I didn't change anything there.

**Pull request merged**